README.md

```diff
@@ -1,4 +1,4 @@
-core-pages
+iron-pages
 ==========
 
-See the [component page](http://polymer-project.org/docs/elements/core-elements.html#core-pages) for more information.
+See the [component page](http://polymer-project.org/docs/elements/iron-elements.html#iron-pages) for more information.
```

Comment:
I've been doing it for new repositories I made. I guess those links will become more useful whenever we push the doc page..